browser/src/index.html

```diff
@@ -1,25 +1,42 @@
-<!doctype html>
+<!DOCTYPE html>
```

Comment:
I'm guessing that most of the changes here were automatic reformatting by your editor or a linter, and the favicon line is the only new part. If that's the case, it's often good, in a situation like this, to have a separate commit where you lint the existing code to make it pretty, before you dig in and make the real changes. That way it's clear what changes actually are functional, as opposed to just aesthetic.

Reply:
However in this case, it's probably OK to leave as-is. Something to keep in mind for next time though.

Reply:
Ohh thats a very good call. I can go back and clean this up later today. Thanks!